pkg/recipes/driver/bicep.go

```diff
@@ -126,11 +126,6 @@ func (d *bicepDriver) Execute(ctx context.Context, opts ExecuteOptions) (*recipe
 		logger.Info("using Azure provider", "deploymentID", deploymentID, "scope", providerConfig.Az.Value.Scope)
 	}
 
-	if opts.Configuration.Simulated {
```

Comment:
We should delete this unit test https://github.com/radius-project/radius/blob/27c1633311f5128e9ac44b143e63761f85f6000a/pkg/recipes/driver/bicep_test.go#L381? Doesn't look like it was doing what we expected it to, since it isn't failing now. :) Same for terraform driver.

Reply:
These tests are actually failing which is good :) But makes sense to remove them now